Contributing.md

```diff
@@ -1,4 +1,8 @@
-# First Time Setup
+# First Time Setup in Eclipse Desktop
+
+Note: Eclipse Desktop IDE is providing the most convenient development workflow for the Camel Language Server. That said, other IDEs can be used.
```

Comment:
I am tempted to say this is a matter of personal bias and taste and hence does not really belong into this contributing guide except you want to definitely drive people to use Eclipse over any other IDE.

Reply:
not completely a matter of taste. There are only 2 IDEs that are providing specific tooling for Language Server edition. VS Code and Eclipse Desktop. In this case, the Language Server is written in Java. Java support for Eclipse Desktop is by far better than VS Code one.
but ok, can tell that it is not "the most" but "one of the most".

Reply:
I am not judging which one is better but we should simply be clear in our message. Do we want to suggest a specific IDE or do we not? If we want to suggest Eclipse, then be clear about it. That is all I wanted to state.

Reply:
I want to make it clear that other IDEs can be used so that users don't think they are stuck with Eclipse.

Reply:
I'd say then this is not really achieving this goal. It reads more like you should use Eclipse but there is an option to use other IDEs ;)

Reply:
I wan't clear in my comments. "I want to make it clear that other IDEs can be used so that users don't think they are stuck with Eclipse." was only in opposition to your previous comment. Not for the whole intent.

More generally, in this contributing guide, I want to make it clear that Eclipse desktop is a very convenient way to develop Camel Language Server. Which is why the contributing guide is written targeting Eclipse Desktop. But it is possible to use another IDE too. Users are not stuck with Eclipse Desktop.

Reply:
what about?

`Eclipse Desktop IDE is providing one of the most convenient development workflow for the Camel Language Server. This is why this guide is focused on using Eclipse Desktop IDE. Other IDEs can be used too as it remains a plain Maven Java project for the server part. The testing with the client part will require more work.`

Reply:
How about `Eclipse Desktop IDE is providing a convenient development workflow for the Camel Language Server.`